sandbox/extract-imagery-metadata_mg.R

```diff
@@ -161,21 +132,27 @@ date <- datetimedf[1, 143]$date
 date
 #now using the newly seperated date column and using the first row to get the date to be used in the function below
 
-#used the centroid calculated in the above code chunk for the lat/long
-solarnoon <- solarnoon(
-  matrix(c(xcoordinate, ycoordinate), nrow = 1),
+#used the general coordinates and input into this function
+
+sncalc <- solarnoon(
+  matrix(c(coordinate), nrow = 1),
   as.POSIXct(date),
   POSIXct.out=TRUE
 )
 
-solarnoon
+sncalc
 #This gives the solar noon as a fraction of the day as well as the datetime.
 
 #END CODE FOR SOLARNOON CALCULATION
 
 
+#START CODE FOR EARLIEST/LATEST TIME
 
+earliest_time_local <- min(exif$capture_datetime)
+earliest_time_local
 
+latest_time_local<- max(exif$capture_datetime)
+latest_time_local
```

Comment:
This looks good but the output needs to be in the format specified in the protocol:

`YYYYMMDD HHMMSS (local time zone, 24 hr)`

Currently it is a POSIXct datetime object that prints as `"2022-06-30 14:32:47 UTC"`, but it needs to be turned into a pure string (character vector) formatted as above.